public/scripts/report.js

```diff
@@ -1,24 +1,80 @@
 (function(){
 
-  // GET LOCATION
-  var locationButton = document.getElementById('getLocationButton');
-
-  function getLocation() {
-    if (navigator.geolocation) {
-      navigator.geolocation.getCurrentPosition(showPosition);
-    } else {
-      // non-supprt error handling
-      console.log('Geolocation is not supported by this browser.');
+  // GET USER'S LOCATION
+  function getLocation(){
+
+    var locationButton = document.getElementById('getLocationButton');
+
+    function getLocation() {
+      if (navigator.geolocation) {
+        navigator.geolocation.getCurrentPosition(storePosition);
```

Comment:
I'm getting an error from getCurrentPosition at the moment - I think it's probably just my browser, but it might be useful adding the [second argument](https://developer.mozilla.org/en-US/docs/Web/API/Geolocation/getCurrentPosition) which handles errors - took me a whole to work out why nothing was happening!